scripts/rollup/banner-plugin.mjs

```diff
@@ -0,0 +1,8 @@
+export default function banner () {
+  return {
+    name: 'banner',
+    renderChunk (code, chunk) {
+      return chunk.name === 'urql-next' ? `"use client"\n\n${code}` : code
```

Comment:
The only thing I'm a little concerned about here is that we're not using Rollup's banner feature. Mainly, there's nothing wrong with not using it, but I suspect this will mess with sourcemaps

After #3201 I'm now even avoiding any transformations (outside of Rollup plugins) to not having to deal with returning modified sourcemaps or having to use `magic-string` for simple changes. But I suspect, just using the Rollup banner feature could maybe help?

https://github.com/urql-graphql/urql/issues/3182 has more information on how I tested sourcemaps to be working in the longer bundles, but I can test this here as well

Reply:
Gotcha, I'll see how the banner option works, for reference https://rollupjs.org/configuration-options/#output-banner-output-footer